### PR TITLE
Add a note that users must have premium SD account.

### PR DESCRIPTION
**Pull request by songy23**

### CODE REVIEW

File: exporters/stats/stackdriver/README.md [hunk 1]
```diff
@@ -94,5 +95,14 @@ StackdriverStatsExporter.createAndRegisterWithProjectId("MyStackdriverProjectId"
...
 
 ## FAQ
+### Why did I get a PERMISSION_DENIED error from Stackdriver when using this exporter?
+To use our Stackdriver Stats exporter, your Stackdriver account needs to have permission to [create
+custom metrics](https://cloud.google.com/monitoring/custom-metrics/creating-metrics), and that 
+requires a [Premium tier Stackdriver account](https://cloud.google.com/monitoring/accounts/tiers#this_request_is_only_available_in_the_premium_tier). 
+Please note that by default all new Stackdrvier accounts are Basic tier. To upgrade to a Premium 
+tier Stackdriver account, follow the instructions [here](https://cloud.google.com/monitoring/accounts/tiers#start-premium).
+
+In addition, if you've enabled Premium tier but still don't have access to custom metrics, please
```
Comment (bogdandrutu) [inline exporters/stats/stackdriver/README.md]: This is covered in the Authentication section.

Comment (songy23) [reply exporters/stats/stackdriver/README.md]: I see, removed.

File: exporters/stats/stackdriver/README.md [hunk 2]
```diff
@@ -94,5 +95,11 @@ StackdriverStatsExporter.createAndRegisterWithProjectId("MyStackdriverProjectId"
 Java 7 or above is required for using this exporter.
 
 ## FAQ
+### Why did I get a PERMISSION_DENIED error from Stackdriver when using this exporter?
+To use our Stackdriver Stats exporter, your Stackdriver account needs to have permission to [create
+custom metrics](https://cloud.google.com/monitoring/custom-metrics/creating-metrics), and that 
+requires a [Premium tier Stackdriver account](https://cloud.google.com/monitoring/accounts/tiers#this_request_is_only_available_in_the_premium_tier). 
+Please note that by default all new Stackdrvier accounts are Basic tier. To upgrade to a Premium
```
Comment (adriancole) [inline exporters/stats/stackdriver/README.md]: Typo in stackdriver

Comment (songy23) [reply exporters/stats/stackdriver/README.md]: Good catch! Post https://github.com/census-instrumentation/opencensus-java/pull/835 for this.

**Comment by songy23** [comment]
@adriancole We're considering adding pre-defined opencensus metrics (e.g RPC client latency) to Stackdriver. Those metrics will be under domain "opencensus.googleapis.com/" instead of "custom.googleapis.com/opencensus/", and even Basic tier Stackdriver users should have permission to upload stats for these metrics. This is tracked in https://github.com/census-instrumentation/opencensus-java/issues/602, and will probably be done in the version.

**Comment by adriancole** [comment]
Cool. Maybe ref that issue in the readme?

**Comment by songy23** [comment]
I'll prefer to leave it as it is, since currently we don't have a solid timeline for supporting that.

**Pull request merged**